README.md

```diff
@@ -1,47 +1,33 @@
-<p align="center">
-<img src="https://github.com/jina-ai/jina/blob/master/.github/logo-only.gif?raw=true" alt="Jina banner" width="200px">
-</p>
-
-# CLIPTextEncoder
+# ✨ CLIPTextEncoder
 
  **CLIPTextEncoder** is a class that wraps the text embedding functionality from the **CLIP** model.
 
-The **CLIP** model was originally proposed in  [Learning Transferable Visual Models From Natural Language Supervision](https://cdn.openai.com/papers/Learning_Transferable_Visual_Models_From_Natural_Language_Supervision.pdf).
+The **CLIP** model was originally proposed in  [Learning Transferable Visual Models From Natural Language Supervision](https://arxiv.org/abs/2103.00020).
 
 `CLIPTextEncoder` encodes data from a `np.ndarray` of strings and returns a `np.ndarray` of floating point values.
 
 - Input shape: `BatchSize `
 
 - Output shape: `BatchSize x EmbeddingDimension`
 
-## Prerequisites
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
 
-To install the dependencies locally run 
-```
-pip install . 
-pip install -r tests/requirements.txt
-```
-To verify the installation works:
-```
```

Comment:
why removing this part? maybe move it to notes or somewhere else if we don't want it in Prerequisites

Reply:
I removed it as this is required for development, but not for just using the executor (according to all the ways that we list - they either already have the requirements installed (docker) or will install them). 

So if I take it out of Prerequisites, there is no place where it would fit - and the template does not have a "Development" section. But it also doesn't fit into Prerequisites, so I removed it.